src/routes/oauth/oauthUser.ts

```diff
@@ -17,4 +21,24 @@ export default class extends Route {
 		return this.client.dashboardUsers.add(user);
 	}
 
+	@authenticated
+	@ratelimit(2, 1000 * 60 * 5, true)
```

Comment:
```suggestion
	@ratelimit(2, 300000, true)
```
Should use constant number for consistency